barretenberg/cpp/src/barretenberg/ultra_honk/oink_prover.cpp

```diff
@@ -26,6 +26,7 @@ template <IsUltraFlavor Flavor> OinkProverOutput<Flavor> OinkProver<Flavor>::pro
     execute_grand_product_computation_round();
 
     return OinkProverOutput<Flavor>{
+        .proving_key = std::move(proving_key),
```

Comment:
not sure if this one is necessary too. Do we need to do 2 std::moves() in order to avoid copying when returning?

Reply:
Hm, my sense is it's better to have these though perhaps the compiler would already move? So yeah, also not sure.